utils/rpms/10-daos_server.conf

```
@@ -0,0 +1,6 @@
+# To override settings in this file, create a local file in /usr/lib
+# (e.g. /usr/lib/sysctl.d/90-override.conf), and put any assignments
+# there.
+
+net.core.rmem_max=104857 +
```

Is the trailing ` +` intended here?

If not:

```suggestion
net.core.rmem_max=104857
```

---

section 2.2.2 of https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/kernel_administration_guide/working_with_sysctl_and_kernel_tunables specifies this

---

I see it their example but don't see any explanation of it on that page.  I wonder if that is a typo in their doc page.  I cannot find anywhere else from a Google search or even the `sysctl.conf` manpage that documents that syntax.

Have you tried applying that file, locally on an EL8 system to see if any syntax/parsing errors are produced?  If not, are the values altered as the file prescribes?  I.e. they start with some other value and end up with the value the file specifies?

Could we expect CI to have failed in any particular way if the values were not changed?

---

Yes I have verified the file parameters get applied correctly on EL8 after RPM install using wolf-16[5-7]. If the parameters get applied correctly we should get no ERROR regarding pci_event from SPDK in the server and engine logs.

---

I [checked with the procps maintainers](https://gitlab.com/procps-ng/procps/-/issues/225) and they confirm that the trailing `+` looks like a RedHat documentation bug:

```
It looks like an issue with RedHat's documentation.
```

[RHEL 8 documentation](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/system_design_guide/configuring-kernel-parameters-at-runtime_system-design-guide#using-configuration-files-in-etc-sysctl-d-to-adjust-kernel-parameters_configuring-kernel-parameters-at-runtime) of the same thing also does not have any use of a `+`.

I really do not think we should have a `+` in that file.

---

removed